scalafix/rules/src/main/scala/fix/Scalaz2cats_v1_0_0.scala

```diff
@@ -223,3 +223,71 @@ case class RemoveGlobalImports(index: SemanticdbIndex)
     }.asPatch
   }
 }
+
+case class MigrateEqual(index: SemanticdbIndex) extends SemanticRule(index, "MigrateEqual") {
+
+  private lazy val scalazEqualSyntaxImport = importer"scalaz.syntax.equal._"
+  private lazy val scalazEqualImport = importer"scalaz.Equal"
+  private lazy val catsEqualSyntaxImport = importer"cats.syntax.eq._"
+  private lazy val catsEqualImport = importer"cats.Eq"
+
+  private lazy val Equal = SymbolMatcher.normalized(
+    Symbol("_root_.scalaz.Equal.")
+  )
+
+  private lazy val catsEqual = SymbolMatcher.normalized(
+    Symbol("_root_.cats.Eq.")
+  )
+
+  private lazy val `===` = SymbolMatcher.normalized(
+    Symbol("_root_.scalaz.syntax.EqualOps.`===`.")
+  )
+  private lazy val `=/=` = SymbolMatcher.normalized(
+    Symbol("_root_.scalaz.syntax.EqualOps.`=/=`.")
+  )
+
+  private lazy val `/==` = SymbolMatcher.normalized(
+    Symbol("_root_.scalaz.syntax.EqualOps.`/==`.")
+  )
+
+  private lazy val `≠` = SymbolMatcher.normalized(
+    Symbol("_root_.scalaz.syntax.EqualOps.`≠`.")
+  )
+
+  private lazy val `≟` = SymbolMatcher.normalized(
+    Symbol("_root_.scalaz.syntax.EqualOps.`≟`.")
+  )
+
+  override def fix(ctx: RuleCtx): Patch = {
+    ctx.tree.collect {
+      case t @ importer"scalaz.syntax.equal._" =>
+        ctx.replaceTree(t, catsEqualSyntaxImport.syntax)
+      case t @ importer"scalaz.Equal" =>
+        ctx.replaceTree(t, catsEqualImport.syntax)
+      case t @ Type.Apply(Equal(_), List(tpe)) =>
+        ctx.replaceTree(t, q"Eq[$tpe]".syntax)
+      case Term.Select(a @ Equal(_), _) =>
+        ctx.replaceTree(a, q"Eq".syntax)
+      case Type.Param(_, _, _, _, _, List(a @ Equal(_))) =>
+        ctx.replaceTree(a, q"Eq".syntax)
+    }.asPatch + ctx.replaceSymbols(
+      "scalaz.syntax.EqualOps.`=/=`" -> "`=!=`",
+      "scalaz.syntax.EqualOps.`/==`" -> "`=!=`",
+      "scalaz.syntax.EqualOps.`≠`" -> "`=!=`",
+      "scalaz.syntax.EqualOps.`≟`" -> "`===`",
```

Comment:
oh gosh, how are you supposed to type this symbol? :O

Reply:
It's just copy-past from scalaz source 😜

Reply:
haha, I know, I'm wondering how many people actually use this when using scalaz